src/components/existing-request/existing-request-display.vue

```diff
@@ -228,6 +262,14 @@ export default class ExistingRequestDisplay extends Mixins(
   /** The pending payment, if any. See mounted(). */
   private pendingPayment = null
 
+  /** Params used to format refund message (tooltip and modal). */
+  private refundParams: RefundParamsIF = {
+    'refundMessage': '',
+    'refundLabel': '',
+    'showStaffContact': false,
+    'showAlertIcon': false
```

Comment:
You don't need the quotes around the property names.

Reply:
👍 done.